getting_started/setup_vm/roles/ccf_build/tasks/install.yml

```diff
@@ -1,3 +1,10 @@
+- name: Add Python repository
```

Comment:
I am confused by this, doesn't 22.04 ship with 3.10? If so, why do we need the ppa?

Reply:
This PR doesn't update the base container image to 22.04, and only updates Python to 3.10. Updating our container images to 22.04 will be a bigger change (in a separate PR) which cannot be completed just yet because of the lack of Open Enclave packages on 22.04.